packages/service-registry-api/src/primitives.rs

```diff
@@ -62,3 +62,21 @@ pub enum AuthorizationState {
     Authorized,
     Jailed,
 }
+
+// Represents any modifiable fields of the Service struct
+// Any non-None field overwrites the value currently stored in the Service object
+#[cw_serde]
+pub struct UpdatedServiceParams {
+    pub min_num_verifiers: Option<u16>,
+    pub max_num_verifiers: Option<Option<u16>>,
+    pub min_verifier_bond: Option<nonempty::Uint128>,
+    pub unbonding_period_days: Option<u16>,
+}
+
+// Represents any overrideable fields of the Service struct
+// Any non-None field overrides the value currently stored in the Service object
+#[cw_serde]
+pub struct ServiceParamsOverride {
+    pub min_num_verifiers: Option<u16>,
+    pub max_num_verifiers: Option<Option<u16>>,
```

Comment:
can we add some logic to actually enforce the max? doesn't necessarily need to be in this PR, but a followup. I don't like that we are adding stuff to the API that actually does nothing.